certbot/tests/cli_test.py

```diff
@@ -352,12 +352,27 @@ def test_option_was_set(self):
         self.assertTrue(cli.option_was_set(key_size_option, key_size_value))
         self.assertTrue(cli.option_was_set('no_verify_ssl', True))
 
+        ecdsa_key_size_option = 'ecdsa_key_size'
+        ecdsa_key_size_value = cli.flag_default(ecdsa_key_size_option)
+        self.parse('--ecdsa-key-size {0}'.format(ecdsa_key_size_value).split())
+
+        self.assertFalse(cli.option_was_set(ecdsa_key_size_option, ecdsa_key_size_value))
+
         config_dir_option = 'config_dir'
         self.assertFalse(cli.option_was_set(
             config_dir_option, cli.flag_default(config_dir_option)))
         self.assertFalse(cli.option_was_set(
             'authenticator', cli.flag_default('authenticator')))
 
+        key_type_option = 'key_type'
+        key_type_value = cli.flag_default(key_type_option)
+        self.parse('--key-type {}'.format(key_type_value))
+        self.assertFalse(cli.option_was_set(key_type_option, key_type_value))
```

Comment:
Same as above, I think `cli.option_was_set` should return `True`.

Reply:
Absolutely correct (same as other) -- i forgot to add a TODO to follow up on the unexpected behavior. Corrected.

Reply:
Done